src/components/Bar/Bar.spec.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Bar from './Bar'
+
+describe('<Bar />', () => {
+  it('should render bar', () => {
+    const { container } = render(<Bar />)
+    const barEl = container.firstChild
+
+    expect(barEl).toBeInTheDocument()
+  })
+
+  it('should handle bar with correct size', () => {
+    const { container, rerender } = render(<Bar size='sm' />)
+    const barEl = container.firstChild
+
+    expect(barEl).toHaveStyleRule('height', '27px')
```

Comment:
wouldn't it be better if we compared rendered height to the helper variable that is used to specify Bar components height?
https://github.com/arturbien/React95/blob/ea317b8618e8fd064db8225c558cda9625cb40af/src/components/Bar/Bar.js#L4
https://github.com/arturbien/React95/blob/ea317b8618e8fd064db8225c558cda9625cb40af/src/components/Bar/Bar.js#L9

Reply:
Totally agree with u! Fixed :)